### ISSUE TITLE
Your requirements could not be resolved to an installable set of packages.

### ISSUE THREAD

**Issue by eaguad1337** [froala/nova-froala-field#16]
When I try `froala/nova-froala-field` I get the following error, can't understand what I am doing wrong.

```
 - Installation request for froala/nova-froala-field ^2.0 -> satisfiable by froala/nova-froala-field[2.0.0].
    - froala/nova-froala-field 2.0.0 requires laravel/nova ^2.0 -> no matching package found.
```

Composer.json

```
{
    "name": "laravel/laravel",
    "description": "The Laravel Framework.",
    "keywords": ["framework", "laravel"],
    "license": "MIT",
    "type": "project",
    "require": {
        "php": ">=7.0",
        "dmitrybubyakin/nova-medialibrary-field": "^1.1",
        "epartment/nova-dependency-container": "^1.1",
        "fideloper/proxy": "^4.0",
        "freshwork/chilean-bundle": "^2.0",
        "guzzlehttp/guzzle": "^6.3",
        "laravel/framework": "5.8.*",
        "laravel/nova": "*",
        "laravel/passport": "^7.0",
        "laravel/tinker": "~1.0",
        "maatwebsite/excel": "^2.1",
        "mpociot/laravel-apidoc-generator": "^2.0",
        "orlyapps/nova-belongsto-depend": "^0.0.8",
        "spatie/laravel-medialibrary": "^7.0.0",
        "spatie/laravel-permission": "^2.6",
        "webpatser/laravel-uuid": "^3.0"
    },
    "require-dev": {
        "doctrine/dbal": "~2.3",
        "filp/whoops": "~2.0",
        "fzaninotto/faker": "~1.4",
        "infyomlabs/adminlte-templates": "5.8.x-dev",
        "infyomlabs/laravel-generator": "5.8.x-dev",
        "laravelcollective/html": "^5.5.0",
        "mockery/mockery": "^1.1",
        "phpunit/phpunit": "~6.0"
    },
    "repositories": [
        {
            "type": "path",
            "url": "./nova"
        }
    ],
    "autoload": {
        "classmap": [
            "database"
        ],
        "psr-4": {
            "App\\": "app/"
        },
        "files": [
            "app/helpers.php"
        ]
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ]
    },
    "minimum-stability": "dev",
    "prefer-stable": true,
    "config": {
        "preferred-install": "dist",
        "sort-packages": true,
        "optimize-autoloader": true
    }
}
```

**Comment by slavarazum** [comment]
**nova-froala-field**@2.0.0 requires:
- **laravel/framework**@5.8
- **laravel/nova**@2.0.0

**Comment by BenSampo** [comment]
I ran into this same issue. It seems you must install Nova via composer in order for composer to be able to pick up the version number correctly.

https://nova.laravel.com/docs/2.0/installation.html#installing-nova-via-composer

Would be good if we could install this package using a local Nova install too.

**Comment by slavarazum** [comment]
Just published **2.1.0** version with ability to local installation. Let me know, if everything works fine.
